README.md

```diff
@@ -0,0 +1,141 @@
+# Rootless Pixel Launcher
+
+![Rootless Pixel Launcher in action](https://i.imgur.com/wTCa3uW.gif)
```

Comment:
Probably better to upload directly to the repo

Reply:
Yes, that's what I suggested amir.

Reply:
you can include it into your pr ;)

Reply:
Found out how to do this.